### PR TITLE
Attempt to use trustify-ui as a crate dependency.

### PR DESCRIPTION
**Pull request by bobmcwhirter**
Pending merging (and un-forking) of https://github.com/trustification/trustify-ui/pull/13

### CODE REVIEW

File: server/src/lib.rs [hunk 1]
```diff
@@ -144,8 +146,20 @@ impl InitData {
...
+
         let storage = DispatchBackend::Filesystem(FileSystemBackend::new(storage).await?);
 
+        // TODO: where/how should we configure these details?
+        let ui = UI {
+            version: env!("CARGO_PKG_VERSION").to_string(),
+            auth_required: String::from("false"),
+            oidc_server_url: String::from("http://localhost:8180/realms/trustify"),
+            oidc_client_id: String::from("trustify-ui"),
+            oidc_scope: String::from("email"),
```
Comment (carlosthe19916) [inline server/src/lib.rs:158]: I believe it should be populated with the values of `AuthConfigArguments` https://github.com/trustification/trustify/blob/main/common/auth/src/auth.rs#L15

But we can do it in another PR

**Review by carlosthe19916 — APPROVED** [review]

**Review by jcrossley3 — APPROVED** [review]
Works really well!!

**Comment by jcrossley3** [comment]
Maybe update the title? No longer forked, correct?

File: server/src/lib.rs (hunk 1 above)
Comment (bobmcwhirter) [reply server/src/lib.rs:158]: Okay, I'll work with @ctron in a subsequent PR.

**Comment by bobmcwhirter** [comment]
> Maybe update the title? No longer forked, correct?

Correct!
